### DISCUSSION TITLE
Option to separate krita_plugin and stable-diffusion-webui?

### DISCUSSION THREAD

**Discussion by a-l-e-x-d-s-9** [sddebz/stable-diffusion-krita-plugin#38] [General]
Is there an option to use stable-diffusion-webui installed on my system, and launch webui.sh from it? And separately run krita_server.py that would connect to the running instance of stable-diffusion?
As far as I understand, currently running krita_server.py will create its own instance stable-diffusion-webui with files and settings.
I don't want to keep duplication of files, models, settings, etc. between the regular stable-diffusion-webui folder and stable-diffusion-krita-plugin. Also, when there are changes in stable-diffusion-webui, it would be great to get them right away.
Is there a way to separate krita-plugin from stable-diffusion-webui?

**Comment by bnelsey** [comment]
this plugin is awesome but I also need features found in the updated version of AUTOMATIC1111's SD webui
support +1

**Reply by Lyssers** [reply]
Same here, the new inpainting model support over at automatic1111's repo is amazing and if it can be integrated into Krita it will be basically perfection, and the endgame solution to that is to have this plugin use the up-to-date main version of WebUI instead of a custom one. Hopefully the plugin code can be sufficiently decoupled from the WebUI/SD code?
